xml/cha_administration.xml

```diff
@@ -28,4 +28,187 @@
+      valid registration code. Replace <replaceable>EMAIL_ADDRESS</replaceable>
+      with the e-mail address associated with the &suse; account you or your
+      organization uses to manage subscriptions.
+     </para>
+     <screen>&prompt.root;<command>registercloudguest</command> -r <replaceable>REGISTRATION_CODE </replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable></screen>
+     <para>
+      This will only register the base product and any recommended products.
+      For instances created from images with a datestamp later than
+      <literal>20220103</literal>, it will also set up the repositories for the
+      <literal>Public Cloud Module</literal>.
+     </para>
+    </step>
+    <step>
+     <para>
+      For instances created from images with a date stamp prior to
+      <literal>20220103</literal> it is strongly recommended to also add the
+      <literal>Public Cloud Module</literal> repositories. For &sle; 15 SP3,
+      run:
+     </para>
+<screen>&prompt.root;<command>&suseconnect;</command> -p sle-module-public-cloud/15.3/x86_64</screen>
+     <para>
+      For other versions or a complete list of modules and their product
+      identifiers, run <command>&suseconnect; --list-extensions</command>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec-admin-suseconnect">
+   <title>Register with <command>&suseconnect;</command></title>
+   <para>
+    To register a BYOS instance against the &scc;, use the command:
+   </para>
+   <screen>&prompt.root;<command>&suseconnect;</command> -r <replaceable>REGISTRATION_CODE</replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable></screen>
+   <para>
+    Replace <replaceable>REGISTRATION_CODE</replaceable> with a valid
+    registration code. Replace <replaceable>EMAIL_ADDRESS</replaceable> with the
+    e-mail address associated with the &suse; account you or your organization
+    uses to manage subscriptions.
+   </para>
+   <para>
+    To register with your own registration server, also provide its URL:
+   </para>
+   <screen>&prompt.root;<command>&suseconnect;</command> -r <replaceable>REGISTRATION_CODE</replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable> --url <replaceable>URL</replaceable></screen>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec-admin-deregister">
+  <title>Deregister instances</title>
+  <para>
+   If you are to decommission an instance, remember to deregister it before
+   termination. This will ensure that the system gets removed from the &scc; and
+   is no longer counted against your subscription.
+  </para>
+  <para>
+   If a system was registered with <command>registercloudguest</command>, run:
+  </para>
+  <screen>&prompt.root;<command>registercloudguest</command> <option>--clean</option></screen>
+  <para>
+   If a system was registered with <command>&suseconnect;</command>, run:
+  </para>
+  <screen>&prompt.root;<command>&suseconnect;</command> <option>-d</option></screen>
```

Comment:
Same as I mentioned before, maybe being more explicit about the commands, if the system was registered with SUSEConnect but registercloudguest is installed, this will *not* work (and it will show a message suggesting to use registercloudguest --clean but we can avoid the user going that path)

Reply:
@jesusbv, that makes sense. We could then consolidate the troubleshooting in one procedure. But this means, we need

- a simple command to get the current registration status (step one)
- a way to determine what tool was used for the registration (step two)

Reply:
So for the registration status, I used `SUSEConnect --status-text`. To determine what tool was used I check for presence of /var/log/guestregister. If the file is present I assume registercloudguest, if not SUSEConnect.

Is this save? Could we run into a corner case, say a system was registered with SUSEConnect, but cloud-regionsrv-client was installed afterwards, something called registercloudguest and created the log file?